CommonCompiler.cmake

```diff
@@ -1,12 +1,13 @@
-# Copyright (c) 2012-2014 Fabien Delalondre <fabien.delalondre@epfl.ch>
+# Copyright (c) 2012-2016 Fabien Delalondre <fabien.delalondre@epfl.ch>
 #                         Stefan.Eilemann@epfl.ch
 #
 # Sets compiler optimization, definition and warnings according to
 # chosen compiler. Supported compilers are XL, Intel, Clang, gcc (4.4
 # or later) and Visual Studio (2008 or later).
 #
-# This defines the common_compile_options() function to apply compiler flags and
-# features for the given target.
+# This defines the common_compile_options() function to apply compiler
+# flags and features for the given target, and adds the target to the
+# project-all meta target.
```

Comment:
great! then this file should be reverted (no changes, comment no longer applies)